components/get_post_link_modal/get_post_link_modal.tsx

```diff
@@ -2,57 +2,73 @@
 // See LICENSE.txt for license information.
 
 import React from 'react';
-import PropTypes from 'prop-types';
 
 import ModalStore from 'stores/modal_store.jsx';
 import Constants from 'utils/constants';
 import * as Utils from 'utils/utils.jsx';
 import GetLinkModal from 'components/get_link_modal';
 
-export default class GetPostLinkModal extends React.PureComponent {
-    static propTypes = {
+type Props = {
+    currentTeamUrl: string;
+}
 
-        /**
-         * URL of current team
-         */
-        currentTeamUrl: PropTypes.string.isRequired,
+type State = {
+    show: boolean;
+    post: {
+        id?: string;
     };
+}
+
+type HandleToggle={
+    (value: boolean, args: any): void;
```

Comment:
I have typed args as `any` is this ok?
If not what type can i refer to it

Reply:
I'm not against the any here (for now) we should convert that any in an specific type in the future.

By the way, would be nice to have the type definition directly where the function is defined, probably is enough to pass the types to the function definition.